styles/components/sankey/node.scss

```diff
@@ -1,41 +1,42 @@
-$someVar: orange;
+@import '../../settings';
 
 .sankey-bg {
   width: 100%;
   height: 100%;
-  fill: white;
+  fill: $white;
 }
 
 .sankey-node {
   cursor: pointer;
 }
 
 .sankey-node-rect {
-  fill: white;
-  stroke: black;
+  fill: $white;
+  stroke: $black;
   stroke-width: .5px;
 }
 
 .sankey-node-label {
   font-size: 12px;
-  text-anchor: middle;
   text-transform: capitalize;
+  text-anchor: middle;
 }
 
 .sankey-link {
   fill: none;
 }
 
 .sankey-clicked-links .sankey-link {
-  stroke: $someVar;
   opacity: .5;
-
+  stroke: $orange;
 }
+
 .sankey-highlighted-links .sankey-link {
-  opacity: 0.2;
-  stroke: red;
+  opacity: .2;
+  stroke: $medium-pink;
 }
+
 .sankey-state-links .sankey-link {
-  opacity: 0.2;
-  stroke: black;
+  opacity: .2;
+  stroke: $black;
 }
```

Comment:
Well, I thought about it but I also thought it was kind of redundant but if you are OK, no problem!